Assets/Scripts/UI/DogAdoptionSlot.cs

```diff
@@ -18,13 +21,31 @@ public override void Init(DogDescriptor dog, Sprite dogSprite, Sprite background
 
         priceOrAdoptionStatus = GetComponentInChildren<Text>();
         priceOrAdoptionStatus.text = dog.CostToAdoptStr;
+
+        priceBackgroundImage = images[2];
     }
 
     #endregion
 
     public void Adopt()
     {
-        priceOrAdoptionStatus.text = "Adopted";
+        if (checkAdoption())
+        {
+            dataController.ChangeCoins(-dog.CostToAdopt);
+            dataController.ChangeOpenHomeSlots(-1);
```

Comment:
This line makes me think we actually want to vars for HomeSlots:

available

*and*

unlocked

So adding / removing dogs can modify the first, and unlocking collars can modify the second